CONTRIBUTING.md

```diff
@@ -64,6 +64,16 @@ Next, to create a new change entry, in the root of the repository run: `changie
 
 Follow the prompts to create your change entry - remember this is what will show up in the changelog.  Changie registers the change in a .yaml file, and that file must be included in your pull request before we can release.
 
+### Auto Generating Types
+
+There is a code generator that helps maintain the various Enum types we use in graphql. It introspects the public Graphql Interface and keeps `enum.go` up to date. To use it:
+
+```
+go run gen.go
```

Comment:
This idiomatic way is

```
go generate
```

If we ever add multiple generators this will ensure to run all of them